cli/cli/commands/run/run.go

```diff
@@ -631,12 +631,18 @@ func parseImageDownloadFlag(flags *flags.ParsedFlags) (*kurtosis_core_rpc_api_bi
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "An error occurred getting the image-download using flag key '%s'", imageDownloadFlagKey)
 	}
-	imageDownload, err := command_args_run.ImageDownloadString(imageDownloadStr)
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "Invalid image-download value: '%s'. Possible values are %s", imageDownloadStr, strings.Join(command_args_run.ImageDownloadStrings(), ", "))
+
+	imageDownloadCode := kurtosis_core_rpc_api_bindings.ImageDownloadMode_value[strings.ToLower(imageDownloadStr)]
+	if imageDownloadCode >= 1 {
```

Comment:
no need to change this but jus mentioning FWIW, often the codebase follows an ["eject early, eject often"](https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/main/general-style-guide.md#default-to-eject-early-eject-often) pattern for conditional branches, just a stylistic choice